relayer/relays/parachain/beefy-listener.go

```diff
@@ -561,6 +563,17 @@ func (li *BeefyListener) scanForCommitments(
 			"blockNumber": currentBlockNumber,
 		}).Debug("Checking header")
 
+		decodedAccount, err := hex.DecodeString(li.config.Account)
+		if err != nil {
+			return nil, fmt.Errorf("could not decode channel account id: %v", err)
+		} else if len(decodedAccount) != 32 {
+			// The conversion below will panic if decodedAccount has
+			// fewer than 32 bytes.
+			// We expect exactly 32 bytes.
+			return nil, fmt.Errorf("account id was not 32 bytes long: %v", decodedAccount)
+		}
+		accountAsArray := *(*[32]byte)(decodedAccount)
```

Comment:
Ah, nice! A new feature I didn't know about. I see now that in Go 1.17 slices can be converted into arrays.